code/modules/antagonists/heretic/knowledge/side_blade_rust.dm

```diff
@@ -73,12 +72,11 @@
 /datum/heretic_knowledge/rifle_ammo
 	name = "Lionhunter Rifle Ammunition"
 	desc = "Allows you to transmute 3 ballistic ammo casings (used or unused) of any caliber, \
-		including shotgun shot, with any animal hide to create an extra clip of ammunition for the Lionhunter Rifle."
+		including shotgun shot to create an extra clip of ammunition for the Lionhunter Rifle."
```

Comment:
It says shotgun 'shot' but the recipe calls for literally any ammo casing. This means anything from a 9mm to a foam dart. Maybe a 'bullet casing' is a better descriptor?

Reply:
1. The description is 
`Allows you to transmute 3 ballistic ammo casings (used or unused) of any caliber, including shotgun shot`
So you're right, 'bullet casing' is a better descriptor... it already uses it 😛 

2. Apparently .310 rounds are caseless. So it can't go infinite

Reply:
1. Ah, yeah I realized it was already mentioning that. Then in that case I think it's probably just unnecessary to mention shotgun shells, I think people would probably already interpret it as valid. Wasn't this at one point 'improvised shot' or something?

2 & 3. I don't believe it has to be empty casings, does it? Empty casings are not a separate type of item. I see the blacklist now but it doesn't have a check for if the bullet is loaded or not as far as I can tell. (it probably shouldn't as that would be arbitrarily more difficult to reload the gun if you needed a separate gun to unload all your bullets with)